### PR TITLE
Added graphics settings for denoising

### PR DESCRIPTION
**Pull request by florianschut**
Also, includes the removal of an unused CB from the denoising task. Aswell as the removal of duplicate namespaces in the settings

### CODE REVIEW

File: tests/demo/scene_sun_temple.hpp [hunk 1]
```diff
@@ -186,7 +186,7 @@ namespace sun_temple_scene
 		fire_bowl_node_2->SetRadius(2000.f);*/
 	}
 
-	void UpdateScene()
+	void UpdateScene(void* scam) //Argument needed to compile...
```
Comment (LaisoEmilio) [inline tests/demo/scene_sun_temple.hpp]: can you rename this, please?

Comment (Nielsbishere) [reply tests/demo/scene_sun_temple.hpp]: lmao

**Comment by Nielsbishere** [comment]
To add additional context: it breaks on DeallocateConstantBuffer, since the handle passed is already nullptr. `for (auto buffer : data.m_denoiser_settings_buffer)`
`{`
`data.m_constant_buffer_pool->Destroy(buffer)`
`}`

**Comment by florianschut** [comment]
all fixed 👌

**Pull request merged**